### PR TITLE
Added Quora Support

### PR DESCRIPTION
**Pull request by DivyanshByte**
Quora Support

Issue: #122

### CODE REVIEW

File: src/sites/quora.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		if (feed == null) {
+			return;
+		}
+
+		const container = feed;
+
+		// Add News Feed Eradicator quote/info panel
+		if (container && !isAlreadyInjected()) {
+			// Hack so that injectUI can handle dark theme
+			// document.body.style.background = 'var(--yt-spec-general-background-a)';
```
Comment (jordwest) [inline src/sites/quora.ts:27]: Can this comment be removed? Looks like copypasta

**Review by jordwest — CHANGES_REQUESTED** [review]
Amazing, thank you!

Can you please remove the diff for `package-lock.json`?

File: src/sites/quora.ts (hunk 1 above)
Comment (DivyanshByte) [reply src/sites/quora.ts:27]: yeah I Just Copied From Youtube One and Modified it :) I will Just Remove It No Problem

**Comment by noethig** [comment]
What happend to this PR? I don't see any settings in the newsFeedEradicator and the newsfeed is not blocked on Quora...

**Pull request closed without merging**